src/pipecat/services/minimax/tts.py

```diff
@@ -97,30 +116,36 @@ class InputParams(BaseModel):
         """Configuration parameters for MiniMax TTS.
 
         Parameters:
-            language: Language for TTS generation.
+            language: Language for TTS generation. Supports 40 languages.
             speed: Speech speed (range: 0.5 to 2.0).
             volume: Speech volume (range: 0 to 10).
             pitch: Pitch adjustment (range: -12 to 12).
             emotion: Emotional tone (options: "happy", "sad", "angry", "fearful",
-                "disgusted", "surprised", "neutral").
-            english_normalization: Whether to apply English text normalization.
+                "disgusted", "surprised", "neutral", "fluent").
+            text_normalization: Enable text normalization (Chinese/English).
+            latex_read: Enable LaTeX formula reading.
+            subtitle_enable: Enable subtitle generation with word-level timestamps.
         """
 
         language: Optional[Language] = Language.EN
         speed: Optional[float] = 1.0
         volume: Optional[float] = 1.0
         pitch: Optional[int] = 0
         emotion: Optional[str] = None
-        english_normalization: Optional[bool] = None
+        text_normalization: Optional[bool] = None
+        latex_read: Optional[bool] = None
+        subtitle_enable: Optional[bool] = None
 
     def __init__(
         self,
         *,
         api_key: str,
-        base_url: str = "https://api.minimax.io/v1/t2a_v2",
+        base_url: str = "https://api.minimax.io/v1/t2a_v2", 
+        # https://api-uw.minimax.io/v1/t2a_v2
```

Comment:
remove the comments here and add them in the comment block like [this](https://github.com/pipecat-ai/pipecat/pull/2981/files#diff-2585adff477c0ce90f47c1317be54b1f99079fec8f684b2923b8f5a679111377R172)